src/go/rdctl/cmd/api.go

```diff
@@ -70,7 +70,7 @@ func doApiCommand(cmd *cobra.Command, args []string) error {
 		return fmt.Errorf("api command: too many endpoints specified (%v); exactly one must be specified", args)
 	}
 	endpoint := args[0]
-	if regexp.MustCompile(`^/v\d+/`).FindString(endpoint) == "" {
+	if endpoint != "/" && regexp.MustCompile(`^/v\d+/`).FindString(endpoint) == "" {
```

Comment:
:cake:
It might be useful to let `rdctl api` / `rdctl api ''` map to `/` instead.
(But that's extending the scope and totally optional; it's fine as-is.)

Reply:
I'd rather reserve them to be errors for now. Sure, we do have a kind of path arithmetic that applies, but it might be wrong to use it now.